packages/orchestration/src/exos/local-chain-account-kit.js

```diff
@@ -147,9 +180,69 @@ export const prepareLocalchainAccountKit = (baggage, makeRecorderKit, zcf) => {
         makeTransferAccountInvitation() {
           throw Error('not yet implemented');
         },
+        /** @type {LocalChainAccount['deposit']} */
+        async deposit(payment) {
+          return E(this.facets.helper.owned()).deposit(payment);
+        },
+        /** @type {LocalChainAccount['withdraw']} */
+        async withdraw(amount) {
+          return E(this.facets.helper.owned()).withdraw(amount);
+        },
+        /**
+         * @returns {ChainAddress['address']}
+         */
+        getAddress() {
+          return NonNullish(this.state.address, 'Chain address not available.');
+        },
+        /**
+         * @param {AmountArg} amount an ERTP {@link Amount} or a {@link DenomAmount}
+         * @param {ChainAddress} destination
+         * @param {IBCMsgTransferOptions} [opts] if either timeoutHeight or timeoutTimestamp are not supplied, a default timeoutTimestamp will be set for 5 minutes in the future
+         * @returns {Promise<void>}
+         */
+        async transfer(amount, destination, opts) {
+          trace('Transferring funds from LCA over IBC');
+          // TODO #9211 lookup denom from brand
+          !matches(amount, AmountShape) || Fail`ERTP Amounts not yet supported`;
```

Comment:
I'm not sure what #9211 should add here. Looking up a denom for a brand in agoricNames.vbankAsset is straightforward.